test/test_utils/__init__.py

```diff
@@ -324,6 +331,47 @@ def get_dlc_images():
     raise RuntimeError(f"Cannot find any images for in {test_images}")
 
 
+def parse_canary_images(framework, region):
+    tf1 = "1.15"
+    tf1_long = "1.15.2"
+    tf2 = "2.2"
+    mx = "1.6"
+    pt = "1.5"
+
+    if framework == "tensorflow":
+        framework = "tensorflow2" if "tensorflow2" in os.getenv("CODEBUILD_BUILD_ID") else "tensorflow1"
```

Comment:
Can you paste sample value of CODEBUILD_BUILD_ID for record?

Reply:
See https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-env-vars.html

```
CODEBUILD_BUILD_ID: The CodeBuild ID of the build (for example, codebuild-demo-project:b1e6661e-e4f2-4156-9ab9-82a19EXAMPLE).
```